lib/fml_forms/form.rb

```diff
@@ -1,6 +1,6 @@
 module FML
   class Form
-    attr_reader :form, :title, :version, :fieldsets, :fields, :bodySystem, :dependent
+    attr_reader :form, :title, :id, :version, :fieldsets, :fields, :bodySystem, :dependent
```

Comment:
it hasn't been since pretty early on in its development, if that's a goal for the future we should move towards it